### DISCUSSION TITLE
v0.3.7

### DISCUSSION THREAD

**Discussion by ilyes319** [ACEsuit/mace#620] [Announcements]
## Main new features

* Finetuning with multiheads : https://mace-docs.readthedocs.io/en/latest/guide/finetuning.html
* Training with multiheads: https://mace-docs.readthedocs.io/en/latest/guide/multihead_training.html

## What's Changed
* Add Intel XPU device  by @jharrymoore in https://github.com/ACEsuit/mace/pull/340
* Intel by @ilyes319 in https://github.com/ACEsuit/mace/pull/356
* Fix logic that deals with IsolatedAtom and 1-atom configuration by @bernstei in https://github.com/ACEsuit/mace/pull/393
* Fix bug that overwrote REF_* keys when those were the explicitly specified keys for the training reference quantities by @bernstei in https://github.com/ACEsuit/mace/pull/403
* Fix check for patience that fails if swa start was not specified by @bernstei in https://github.com/ACEsuit/mace/pull/405
* fix incorrect conversion of forces from calculator to atoms.arrays for finetuning pbe mp replay head by @bernstei in https://github.com/ACEsuit/mace/pull/409
* Better fix for PR #405, fix patience check when swa is not active by @bernstei in https://github.com/ACEsuit/mace/pull/411
* fix np.random.choice issue when subselecting from pre-trained configs for multihead fine tune by @bernstei in https://github.com/ACEsuit/mace/pull/448
* validation loss printed during training is not actually for the specific head by @bernstei in https://github.com/ACEsuit/mace/pull/449
* get rid of all stress/n_atoms by @bernstei in https://github.com/ACEsuit/mace/pull/443
* Stage two weights by @vue1999 in https://github.com/ACEsuit/mace/pull/522
* remove mypy from checks by @ilyes319 in https://github.com/ACEsuit/mace/pull/529
* Fix MAE table for universal loss by @ilyes319 in https://github.com/ACEsuit/mace/pull/548
* Create py.typed by @Andrew-S-Rosen in https://github.com/ACEsuit/mace/pull/554
* Improve Log Details by @vue1999 in https://github.com/ACEsuit/mace/pull/553
* Backward the log changes by @ilyes319 in https://github.com/ACEsuit/mace/pull/571
* Merge develop, pylint + log changes by @ilyes319 in https://github.com/ACEsuit/mace/pull/572
* Multihead finetuning and training by @ilyes319 in https://github.com/ACEsuit/mace/pull/575
* multihead finetuning by @ilyes319 in https://github.com/ACEsuit/mace/pull/578
* Fix bug about undefined swa by @Hongyu-yu in https://github.com/ACEsuit/mace/pull/584
* Fix for Issue 596 by @stenczelt in https://github.com/ACEsuit/mace/pull/597
* Split `download_mace_mp_checkpoint` out of `mace_mp` by @janosh in https://github.com/ACEsuit/mace/pull/602
* Allow creation of MACECalculator without needing to write a checkpoint to disk. by @CompRhys in https://github.com/ACEsuit/mace/pull/535
* fix swa bug and remove e3nn fixed version by @ilyes319 in https://github.com/ACEsuit/mace/pull/589
* Change type hint syntax in MACECalculator to work with python39 by @bernstei in https://github.com/ACEsuit/mace/pull/618
* fix swa bigger than epoch by @ilyes319 in https://github.com/ACEsuit/mace/pull/616
* add option to return raw model in mace_mp by @ilyes319 in https://github.com/ACEsuit/mace/pull/619

## New Contributors
* @Hongyu-yu made their first contribution in https://github.com/ACEsuit/mace/pull/584
* @CompRhys made their first contribution in https://github.com/ACEsuit/mace/pull/535

**Full Changelog**: https://github.com/ACEsuit/mace/compare/v0.3.6...v0.3.7

<hr /><em>This discussion was created from the release <a href='https://github.com/ACEsuit/mace/releases/tag/v0.3.7'>v0.3.7</a>.</em>

**Comment by nihil39** [comment]
Thanks for the new version! 
Is it normal that in the pip repository (I don't know how to call it) the latest version is still 0.3.6?

https://pypi.org/project/mace-torch/

**Comment by ilyes319** [comment]
Hey @nihil39, I still need to fix some bugs flagged by @bernstei before merging. Hopefully will happen soon. Please use the main branch and report any bug you find.

**Comment by JaGeo** [comment]
@ilyes319 It would be great to have 0.3.7 on pypi available soon.
